sdk/communication/Azure.Communication.CallAutomation/src/Models/Events/RecognizeCompleted.cs

```diff
@@ -32,7 +32,36 @@ internal RecognizeCompleted(string callConnectionId, string serverCallId, string
             RecognitionType = recognitionType;
             CollectTonesResult = collectTonesResult;
             ChoiceResult = choiceResult;
+        }
+
+        /// <summary> Initializes a new instance of RecognizeCompleted. </summary>
+        /// <param name="callConnectionId"> Call connection ID. </param>
+        /// <param name="serverCallId"> Server call ID. </param>
+        /// <param name="correlationId"> Correlation ID for event to call correlation. Also called ChainId for skype chain ID. </param>
+        /// <param name="operationContext"> Used by customers when calling mid-call actions to correlate the request to the response event. </param>
+        /// <param name="resultInformation"> Contains the resulting SIP code/sub-code and message from NGC services. </param>
+        /// <param name="recognitionType">
+        /// Determines the sub-type of the recognize operation.
+        /// In case of cancel operation the this field is not set and is returned empty
+        /// </param>
+        /// <param name="recognizeResult"> Defines the result for RecognitionType = Dtmf. </param>
+        public RecognizeCompleted(string callConnectionId, string serverCallId, string correlationId, string operationContext, ResultInformation resultInformation, CallMediaRecognitionType recognitionType, RecognizeResult recognizeResult)
+        {
+            CallConnectionId = callConnectionId;
+            ServerCallId = serverCallId;
+            CorrelationId = correlationId;
+            OperationContext = operationContext;
+            ResultInformation = resultInformation;
+            RecognitionType = recognitionType;
             ReasonCode = new ReasonCode(resultInformation.SubCode.ToString());
+            if (recognizeResult is CollectTonesResult collecToneResult)
```

Comment:
Nit: seems like a typo collectToneResult ?

Reply:
good catch, will fix it